src/javascript/app/base/header.js

```diff
@@ -350,30 +394,40 @@ const Header = (() => {
+
         // Account Switcher Event
-        const acc_switcher                = getElementById('acc_switcher');
-        const account_switcher            = getElementById('account__switcher');
-        const account_switcher_dropdown   = getElementById('account__switcher-dropdown');
+        const acc_switcher                = Client.hasWalletsAccount() ? getElementById('wallet_switcher') : getElementById('acc_switcher');
+        const account_switcher            = Client.hasWalletsAccount() ? getElementById('wallet__switcher') : getElementById('account__switcher');
+        const account_switcher_dropdown   = Client.hasWalletsAccount() ? getElementById('wallet__switcher-dropdown') : getElementById('account__switcher-dropdown');
+        const account_switcher_active     = Client.hasWalletsAccount() ? 'wallet__switcher-dropdown--show' : 'account__switcher-dropdown--show';
+        const wallet_switcher_close       = getElementById('wallet__switcher-close');
         const acc_expand                  = getElementById('header__acc-expand');
-        const account_switcher_active     = 'account__switcher-dropdown--show';
         const current_active_login        = Client.get('loginid');
         const all_login_ids               = Client.getAllLoginids();
         const has_real_account            = all_login_ids.some((loginid) => !/^VRT/.test(loginid));
         const is_virtual                  = current_active_login && current_active_login.startsWith('VRTC');
         const add_account_text_normal     = document.getElementById('add-account-text-normal');
         const add_account_text_eu_country = document.getElementById('add-account-text-eu');
-        const showAccountSwitcher         = (should_open) => {
+        const showAccDropdown         = (should_open) => {
             if (should_open) {
                 account_switcher_dropdown.classList.add(account_switcher_active);
                 acc_expand.classList.add('rotated');
-                $('#acc_tabs').tabs({ active: is_virtual ? 1 : 0 });
-                if (isEuCountry()) {
-                    add_account_text_normal.style.display                   = 'none';
-                } else {
-                    add_account_text_eu_country.style.display               = 'none';
-                }
-                if (isEuCountry() && has_real_account) {
-                    add_account_text_eu_country.parentElement.style.display = 'none';
+
+                if (!Client.hasWalletsAccount()) {
+                    $('#acc_tabs').tabs({ active: is_virtual ? 1 : 0 });
+                    if (isEuCountry()) {
+                        add_account_text_normal.style.display                   = 'none';
+                    } else {
+                        add_account_text_eu_country.style.display               = 'none';
+                    }
+                    if (isEuCountry() && has_real_account) {
```

Comment:
I know, it's an old code and you didn't touch this part, but maybe:
```
if (isEuCountry()) {
   add_account_text_normal.style.display = 'none';
   if (has_real_account) {
     add_account_text_eu_country.parentElement.style.display = 'none';
   }
} else {
    add_account_text_eu_country.style.display = 'none';
}
```

Reply:
lets not touch this lol. Any fart in the wind and the whole account switcher will break. It has like 4 different account types that is juggling lol